packages/sol-compiler/src/solc_wrapper_v06.ts

```diff
@@ -22,4 +22,15 @@ export class SolcWrapperV06 extends SolcWrapperV05 {
         solcInstance.compileStandardWrapper = solcInstance.compile;
         return compileSolcJSAsync(solcInstance, input);
     }
+
+    protected _normalizeOutput(output: StandardOutput): StandardOutput {
+        const _output = super._normalizeOutput(output);
+        // Filter out 'receive' ABI item types until ethers supports it.
+        for (const contracts of Object.values(_output.contracts)) {
+            for (const contract of Object.values(contracts)) {
+                contract.abi = contract.abi.filter(v => v.type !== 'receive');
```

Comment:
If we don't filter these out, `ethers.js` will spam the console with `WARNING`s. Waiting for a fix in the meantime, but it shouldn't make a difference if we leave it out for now.

Reply:
You mean add a longer comment?

Reply:
Ah nvm, misread and thought there were two separate issues. Inline comment looks good.